mypy/version.py

```diff
@@ -1,17 +1,30 @@
 import os
+
 from mypy import git
 
 # Base version.
 # - Release versions have the form "0.NNN".
 # - Dev versions have the form "0.NNN+dev" (PLUS sign to conform to PEP 440).
 # - For 1.0 we'll switch back to 1.2.3 form.
-__version__ = '2.0.0+dev'
+__version__ = '0.940+dev'
 base_version = __version__
```

Comment:
i assume `base_version` is not to be confused with `based_version`? perhaps one of them can be renamed?

Reply:
`base_version` doesn't even exist at install time so idk.